controllers/argo.go

```diff
@@ -144,6 +143,18 @@ func newApplicationValues(p api.Pattern) string {
 	return s
 }
 
+// This function looks at the map entries read
+// from the patterns config map for the ManualSync
+// entry.
+func commonSyncPolicyFromConfigMap() bool {
+	var value bool = true
+
+	if configValueWithDefault(PatternsOperatorConfig, "gitops.ManualSync", GitOpsDefaultManualSync) == "false" {
```

Comment:
@mbaldessari WDYT?  Should this be global or per-pattern?

Reply:
I think things are a bit confusing here at least for me. Before this PR, GitOpsConfig contains the channel, source, csv, use csv and manual approval which all need to end up in the configmap since they all relate to gitops installation. So far so good. This field ManualSync has nothing to do with the gitops installation but relates to the sync option for the cluster-wide argo pattern application. So I think the manualSync *has* to be per pattern by definition. So I'd either (A) leave it in GitOpsConfig and leave things as they are *or* (B) we just drop GitOpsConfig entirely and move the manualSync field inside the PatternSpec. I think I favour (A) because in the future we might expand con customizations of the gitops stuff specifically.

If we agree that this should be per pattern, then we probably want to drop this function and its callers and just read it like before from the CR

Reply:
Agreed on leaving it the way it is for now.

Reply:
I believe what @mbaldessari is saying here, is that if multiple patterns are on a cluster (lets pretend), then they need to be able to use different manualSync values.

Reply:
We decided to include the GitOpsConfig structure with ManualSync in the CRD.  Updates have been made to the code base.